src/cli.js

```diff
@@ -8,10 +7,12 @@ import { start } from './start/start.cli';
 import { testcli } from './testcli/test.cli';
 import { config } from './config/config.cli';
 import { version } from './../package.json';
+import { setTemplate } from './utils/templates/replacements';
 
 const chalk = require('chalk');
 
 export function cli(args) {
+  setTemplate(args.template);
```

Comment:
I think this line could be execute inside generate options because it is going to be used just there. what do you think?

```suggestion
    case 'g':
      cd('_hjs');
      setTemplate(args.template);
      generateCli(args);
      break;
```